src/core/nexus/HY_PointHydroNexus.cpp

```diff
@@ -141,7 +143,9 @@ void HY_PointHydroNexus::add_upstream_flow(double val, long catchment_id, time_s
 
         id_flow_vector v;
         v.push_back(std::pair<long,double>(catchment_id,val));
-         upstream_flows[t] = v;
+        upstream_flows[t] = v;
+
+        std::cerr << "Creating upstream flows for vector for time step " << t << " with single value of " << val << "\n";
```

Comment:
I don't think we want to include this print.  At some point we can disscuss logging capabilities where this might be useful, but for now I would remove or comment this out.

Reply:
Missed that in removing the debugging code.